CHANGES.rst

```diff
@@ -1,8 +1,10 @@
 Changelog
 =========
 
-3.17 (unreleased)
------------------
+4.0 (unreleased)
+----------------
+
+- Drop support for Python 2.7, 3.5, 3.6.
```

Comment:
Could you please also mention the removal of the deprecated `Surrogate` class?

Reply:
Done